HW3.py

```diff
@@ -0,0 +1,9 @@
+norway_text = ('Automatisering akselererer %syeblikket da roboter vil '
```

Comment:
D100 Missing docstring in public module

Usually here better to place full hometask descritpion + your notes.

Example:
```
"""
Hometask 03 String format.

we have norway text in old style formatting
re-write the same text as:
#1 string with format() call
#2 f-string
use linter(https://github.com/wemake-services/wemake-python-styleguide)
to check your new created python module for possible linter errors
try to run code from pycharm/command line
"""
```